docs/ref/csrf.txt

```diff
@@ -211,7 +211,7 @@ The error page, however, is not very friendly, so you may want to provide your
 own view for handling this condition.  To do this, simply set the
 :setting:`CSRF_FAILURE_VIEW` setting.
 
-CSRF failures are logged as warnings to the :ref:`django-request-logger`
+CSRF failures are logged as warnings to the :ref:`django-security-logger` ``csrf``
```

Comment:
Use:
```
:ref:`django.security.csrf <django-security-logger>`
```
and add a `.. versionchanged:: 1.11` annotation below as described at https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features